pkgs/applications/networking/browsers/surf/default.nix

```diff
@@ -13,13 +14,21 @@ stdenv.mkDerivation rec {
     sha256 = "07cmajyafljigy10d21kkyvv5jf3hxkx06pz3rwwk3y3c9x4rvps";
   };
 
-  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook makeWrapper ];
   buildInputs = [ glib glib-networking gsettings-desktop-schemas gtk libsoup webkitgtk ];
 
   inherit patches;
 
   installFlags = [ "PREFIX=$(out)" ];
 
+  # Add run-time dependencies to PATH. Append them to PATH so the user can
+  # override the dependencies with their own PATH.
+  postInstall = let
```

Comment:
We're actually getting two wrappers here because we're already using `wrapGAppsHook`
See https://hydra.nixos.org/build/102641625/download/1/nixpkgs/manual.html#ssec-gnome-hooks `preFixup` example at the bottom of that section.

Reply:
Thanks! I made some changes, do they look right now?